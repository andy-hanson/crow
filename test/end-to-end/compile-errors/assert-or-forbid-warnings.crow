main void()
	assert true : throw "true is not true"
	guard true : throw "true is not true"
