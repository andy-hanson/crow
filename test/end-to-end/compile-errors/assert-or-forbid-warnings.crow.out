exit code: 0
stderr:
test/end-to-end/compile-errors/assert-or-forbid-warnings.crow 2:19-2:24 The expression after the ':' for an assert or forbid is always thrown; it doesn't need 'throw'.
test/end-to-end/compile-errors/assert-or-forbid-warnings.crow 3:5-3:10 Instead of throwing from a conditional expression, use 'assert' or 'forbid'.
